src/core/thread/mle.cpp

```diff
@@ -788,15 +788,20 @@ ThreadError Mle::AppendLeaderData(Message &aMessage)
     return aMessage.Append(&mLeaderData, sizeof(mLeaderData));
 }
 
+void Mle::FillNetworkDataTlv(NetworkDataTlv &aTlv, bool aStableOnly)
```

Comment:
It's OK, but a little bit tight for the two I guess. I will modify as your suggestion. I think we could figure out a better after we provide a shared base class for all TLV used by thread. I noticed classes Thread::TheadTlv, Thread::Mle::Tlv, Thread::MeshCoP::Tlv, Thread::NetworkData::NetworkDataTlv are very similar.